dlrover/python/elastic_agent/torch/training.py

```diff
@@ -629,7 +632,10 @@ def _invoke_run(self, role: str = DEFAULT_ROLE) -> RunResult:
                 logger.error(f"The worker fails with {run_result.failures}")
                 self._report_failure_to_master(run_result.failures)
                 self._save_ckpt_to_storage()
-                if self._remaining_failovers > 0:
+                if (
+                    self._remaining_failovers > 0
```

Comment:
Should put the 'self._remaining_failovers > 0' logic into 'shoud_relaunch_worker' function.